projects.go

```diff
@@ -328,6 +328,7 @@ func (s *ProjectsService) GetProjectEvents(pid interface{}, opt *GetProjectEvent
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/projects.html#create-project
 type CreateProjectOptions struct {
+	UserID                                    *int             `url:"user_id,omitempty" json:"omitempty,omitempty"`
```

Comment:
`UserID` should not be in here. It's a required field that is in the function signature already.

Reply:
hm - I think it's not. You can only use it, if you want to set the owning user of the project. But to be honest, it had no effect on the project, when I tried it yesterday. Thought I still add it for being "feature-complete" regarding the API. The method signature has no such field:

    func (s *ProjectsService) CreateProject(opt *CreateProjectOptions, options ...OptionFunc) (*Project, *Response, error)

Reply:
you are right - I used the attributes for "Create project for user". I'll update the PR